### PR TITLE
livecoin: fetchOrder() implemented

### PR DESCRIPTION
**Pull request by egorFiNE**

### CODE REVIEW

File: js/livecoin.js [hunk 1]
```diff
@@ -377,49 +378,97 @@ module.exports = class livecoin extends Exchange {
...
+
     parseOrder (order, market = undefined) {
-        let timestamp = this.safeInteger (order, 'lastModificationTime');
-        if (!timestamp)
-            timestamp = this.parse8601 (order['lastModificationTime']);
+        let timestamp = undefined;
+        let datetime = undefined;
+        if ('lastModificationTime' in order) {
+            timestamp = this.safeInteger (order, 'lastModificationTime');
+            if (!timestamp) {
```
Comment (kroitor) [inline js/livecoin.js]: Shouldn't this line be like

```
if (timestamp) { // if it is defined
   timestamp = this.parse8601 (timestamp);
}
```

? ;)

Comment (kroitor) [reply js/livecoin.js]: I guess, the correct code for it is:

```JavaScript
let timestamp = this.safeString (order, 'lastModificationTime')
if (typeof timestamp !== 'undefined') {
    if (timestamp.indexOf ('T') >= 0) {
        timestamp = this.parse8601 (timestamp);
    } else {
        timestamp = this.safeInteger (order, 'lastModificationTime');
    }
}
```

How does it sound to you?

**Comment by egorFiNE** [comment]
Fixed.

File: js/livecoin.js [hunk 2]
```diff
@@ -377,49 +378,101 @@ module.exports = class livecoin extends Exchange {
...
+            if (order['type'].indexOf ('BUY') >= 0) {
+                side = 'buy';
+            }
         }
-        if (order['type'].indexOf ('SELL') >= 0) {
-            side = 'sell';
-        } else {
-            side = 'buy';
+        let price = this.safeFloat (order, 'price');
+        let cost = this.safeFloat (order, 'commissionByTrade'); // commission_rate?
```
Comment (kroitor) [inline js/livecoin.js]: The `cost` of an order is:
- `if (status === 'open' and filled === 0) { amount * price }`
- `if (status === 'closed' || status === 'canceled') { filled * price }`

The `cost` of a trade is `amount * price`.

So, an order `cost` or a trade `cost` means the total *quote* volume of the order (whereas `amount` is the *base* volume). The `cost` field itself is there mostly for convenience and can be deduced from other fields.

I'll add that to the Manual )

As for the unified `fee` sub-structure, it should be like this:

```
'fee': {
    'cost': 123.45,
    'rate': 0.001,
    'currency': 'ETH',
}
```

**Comment by egorFiNE** [comment]
@kroitor Fixed. Please review if I have got it right.

**Comment by kroitor** [comment]
> Please review if I have got it right.

Almost ) But no worries, I'll merge it soon and you'll see the edits I made form there. Thx so much!

**Pull request merged**